### PR TITLE
fixing the server

### PR DESCRIPTION
**Pull request by wjcunningham7**
- [ ] I have added the tests to cover my changes.
- [x] I have updated the documentation, VERSION, and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by wjcunningham7** [comment]
watch the logs when testing -- sometimes I see this:

```
Traceback (most recent call last):
  File "/home/will/projects/agnostiq/covalent/covalent_ui/app.py", line 150, in <module>
    socketio.run(app, debug=debug, host="0.0.0.0", port=port, use_reloader=reload)
  File "/home/will/miniconda3/envs/covalent/lib/python3.8/site-packages/flask_socketio/__init__.py", line 628, in run
    run_with_reloader(run_server, **reloader_options)
  File "/home/will/miniconda3/envs/covalent/lib/python3.8/site-packages/werkzeug/_reloader.py", line 418, in run_with_reloader
    ensure_echo_on()
  File "/home/will/miniconda3/envs/covalent/lib/python3.8/site-packages/werkzeug/_reloader.py", line 398, in ensure_echo_on
    termios.tcsetattr(sys.stdin, termios.TCSANOW, attributes)
termios.error: (5, 'Input/output error')
```

and I'm not sure if it's related to these changes.

File: covalent_dispatcher/_cli/service.py [hunk 1]
```diff
@@ -271,13 +266,11 @@ def restart(ctx, port: bool, develop: bool) -> None:
...
     port = port or get_config("user_interface.port")
-    if pid == -1 or port != get_config("user_interface.port") or develop:
-        ctx.invoke(stop)
-        ctx.invoke(start, port=port, develop=develop)
-    elif pid != -1:
-        started = _graceful_restart(UI_PIDFILE)
-        if not started:
-            ctx.invoke(start, port=port, develop=develop)
+
+    print("Invoking stop/start")
```
Comment (FyzHsn) [inline covalent_dispatcher/_cli/service.py:270]: Was this line meant to be here?

**Review by FyzHsn — APPROVED** [review]
Left a minor non-blocking comment.

Tried covalent-restart, it worked without an issue. Checked that the appropriate server was shutdown and restarted.

Didn't see anything in the log file.

One quick question: Before it would say in the log file that the server had started or shut down. Now it doesn't. Is that intended?

**Comment by FyzHsn** [comment]
@wjcunningham7 I find that `covalent stop` does not stop everything running on port 48008. And when followed by `covalent start`, it choses the next available port. 

Screen Shot 2022-03-14 at 5 18 10 PM

**Review by scottwn — APPROVED** [review]

**Pull request merged**